src/main/java/services/impl/FormRecordProcessorImpl.java

```diff
@@ -112,6 +139,13 @@ public static void purgeCases(UserSqlSandbox sandbox) {
         log.info(String.format(
                 "Purged [%d Case, %d Ledger] records in %dms",
                 removedCaseCount, removedLedgers, taken));
-
+        datadogStatsDClient.recordExecutionTime(
+                Constants.DATADOG_TIMINGS,
+                taken,
+                "domain:" + restoreFactory.getDomain(),
+                "username" + restoreFactory.getWrappedUsername(),
+                "request:" + "case-purge",
+                "case-count" + removedCaseCount
```

Comment:
I think there should be a `:` here, but also, I'm not sure what we get out of tracking this here. Is there a particular graph you're thinking of? In general, I think tags are really only useful with discreet values, since the way you use them is always essentially "where <tag> = <value>" or "group by <tag> value". I think if you have something in mind with this, we could bucket it (<10, <100, <1000, <10000, >10000, with tag value names that are in alphabetical order so that it splits them nicely), but if you don't have a particular thing you want to track with this, I'd just remove it.